webidl2js-wrapper.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
+module.exports = require("./lib/XMLSerializer.js");
```

Comment:
Unfortunately haven't kept up with this in a while - what's the intended use for this now?

Reply:
This is related to the WebIDL2JS Constructor and Prototype reform (https://github.com/jsdom/webidl2js/issues/133). The goal of this work is to be able to create for each exposed interface a brand new constructor and prototype chain per realm. For packages consumed by jsdom exposing interfaces, we have settled on using a naming convention (`wedidl2js-wrapper.js`) to import the generated interfaces.

Here is how the DOMException interface is no integrated into jsdom: https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/interfaces.js#L13

Reply:
It would have been kind of nice if we could have configured webidl2js to output this into the root directly. That forces `utils.js` to be generated in the root too, though, which is subpar.